x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/sample_logs_input.test.tsx

```diff
@@ -304,5 +304,30 @@ describe('SampleLogsInput', () => {
         });
       });
     });
+
+    describe('when the file is too large', () => {
+      beforeEach(async () => {
+        // Simulate large log content that would cause a RangeError
+        jest.spyOn(JSON, 'parse').mockImplementation(() => {
+          throw new RangeError();
+        });
+
+        await changeFile(input, new File(['...'], 'test.json', { type }));
+      });
+
+      afterEach(() => {
+        jest.clearAllMocks();
+      });
```

Comment:
NIT: this is not necessary, the `jest.clearAllMocks()` is already called by the `beforeEach` defined in line 148.